x-pack/test/functional/apps/ml/anomaly_detection/lens_to_ml.ts

```diff
@@ -0,0 +1,222 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export default function ({ getService, getPageObject, getPageObjects }: FtrProviderContext) {
+  const ml = getService('ml');
+  const dashboardPanelActions = getService('dashboardPanelActions');
+  const testSubjects = getService('testSubjects');
+  const browser = getService('browser');
+  const retry = getService('retry');
+  const headerPage = getPageObject('header');
+  const PageObjects = getPageObjects(['common', 'timePicker', 'dashboard']);
+  const kibanaServer = getService('kibanaServer');
+  const esArchiver = getService('esArchiver');
+
+  const dashboardTitle = 'lens_to_ml';
+
+  async function retrySwitchTab(tabIndex: number, seconds: number) {
+    await retry.tryForTime(seconds * 1000, async () => {
+      await browser.switchTab(tabIndex);
+    });
+  }
+
+  async function setFarequoteTimerange() {
+    await PageObjects.timePicker.setAbsoluteRange(
+      'Feb 7, 2016 @ 00:00:00.000',
+      'Feb 11, 2016 @ 23:59:54.000'
+    );
+  }
+
+  async function dashboardPreparation(selectedPanelTitle: string) {
+    await PageObjects.dashboard.loadSavedDashboard(dashboardTitle);
+    await ml.dashboardEmbeddables.assertDashboardPanelExists(selectedPanelTitle);
+
+    await setFarequoteTimerange();
+
+    const header = await dashboardPanelActions.getPanelHeading(selectedPanelTitle);
+    await dashboardPanelActions.openContextMenuMorePanel(header);
+  }
+
+  async function createJobInWizard(
+    jobId: string,
+    splitField?: string,
+    aggAndFieldIdentifier?: string
+  ) {
+    await headerPage.waitUntilLoadingHasFinished();
+
+    if (splitField !== undefined) {
```

Comment:
Based on our slack discussion:`if there is a split in the data or two metrics selected`, it should open the multi metric wizard, otherwise it's the single metric wizard. This makes me think that we should validate that the correct wizard is open first, e.g. via
```
if ("there's a split or two metric selected") { // replace pseudo code
  await ml.jobTypeSelection.assertMultiMetricJobWizardOpen();
} else {
  await ml.jobTypeSelection.assertSingleMetricJobWizardOpen();
}
```